smart-py/objkt_swap_v3_test.py

```diff
@@ -710,173 +691,25 @@ def test_pause_collects():
-
-    # try to mint an objkt with too few royalties
-    # TODO should fail but passes currently
-    # scenario += marketplaceV1.mint_OBJKT(
-    #     address = artist1.address,
-    #     amount = 2,
-    #     royalties = 5,
-    #     metadata = sp.bytes('0x697066733a2f2f516d61794e7a7258547a354237577747574868314459524c7869646646504676775a377a364b7443377268456468')
-    # ).run(
-    #     sender = artist1.address,
-    #     valid = False
-    # )
-
-    # objkt number must not have changed
-    scenario.verify(marketplaceV1.data.objkt_id == 152)
-
-    # try to mint without metadata and fail
-    #
-    # TODO null metadata should fail but passes currently
-    #
-    # scenario += marketplaceV1.mint_OBJKT(
-    #     address = artist1.address,
-    #     amount = 1,
-    #     royalties = 200,
-    #     metadata = sp.bytes('0x00')
-    # ).run(
-    #     sender = artist1.address,
-    #     valid = False
-    # )
-
-    # objkt number must not have changed
-    scenario.verify(marketplaceV1.data.objkt_id == 152)
-
-@sp.add_test(name="Test swap v3 failure conditions")
+@sp.add_test(name="Test swap failure conditions")
 def test_swap_failure_conditions():
     # Get the test environment
     testEnvironment = get_test_environment()
     scenario = testEnvironment["scenario"]
     admin = testEnvironment["admin"]
     artist1 = testEnvironment["artist1"]
     artist2 = testEnvironment["artist2"]
-    collector1 = testEnvironment["collector1"]
-    collector2 = testEnvironment["collector2"]
     objkt = testEnvironment["objkt"]
     marketplaceV1 = testEnvironment["marketplaceV1"]
     marketplaceV3 = testEnvironment["marketplaceV3"]
 
-    # fail to mint an OBJKT on behalf of someone else
-    scenario += marketplaceV1.mint_OBJKT(
-        address=artist1.address,
-        amount=1,
-        metadata=sp.pack("ipfs://fff"),
-        royalties=100
-    ).run(
-        valid=False,
-        sender=artist2
-    )
-
-    scenario.verify(marketplaceV1.data.objkt_id == 152)
-
-    # fail to mint an OBJKT on behalf of someone else as admin
-    scenario += marketplaceV1.mint_OBJKT(
-        address=artist1.address,
-        amount=1,
-        metadata=sp.pack("ipfs://fff"),
-        royalties=100
-    ).run(
-        valid=False,
-        sender=admin
-    )
-
-    scenario.verify(marketplaceV1.data.objkt_id == 152)
-
     # Mint an OBJKT
     scenario += marketplaceV1.mint_OBJKT(
         address=artist1.address,
         amount=1,
         metadata=sp.pack("ipfs://fff"),
-        royalties=100
-    ).run(
-        sender=artist1
-    )
-
-    scenario.verify(marketplaceV1.data.objkt_id == 153)
-
-    # Try to Swap the OBJKT but fail because artist is not admin
-    scenario += marketplaceV3.swap(
-        fa2=objkt.address,
-        objkt_id=152,
-        objkt_amount=1,
-        xtz_per_objkt=sp.mutez(10000),
-        royalties=100,
-        creator=artist1.address
-    ).run(
-        valid=False,
-        sender=artist1
-    )
```

Comment:
this test can remain

Reply:
This test was wrong. It fails because it didn't give the operators right to the contract